packages/TelegramClient-UI.package/TCUUserList.class/instance/addUserBack..st

```diff
@@ -1,6 +1,9 @@
 drawing
 addUserBack: aUser
 
-	self content addMorphBack: (TextMorph new
+	self scroller addMorphBack: (TextMorph new
```

Comment:
please break out `addMorphBack:` onto a new line

Reply:
maybe also indent contents of Textmorph once more

Reply:
We think that this is not necessary because `addMorphBack:` is neither in a cascade, nor a message with more than one parameter. So we think the current formatting should be correct and consistent with the rest.